index.js

```diff
@@ -1314,6 +1314,8 @@ function Runtime() {
         });
 
         return function (target, data) {
+            if (target.isNull()) 
+                throw new Error("The target protocol is null");
```

Comment:
I mean this will crash if you pass `ObjC.Object`, which is a supported use-case. The next line checks the type, but this line does not, so it will implicitly break the functionality provided by the next line.